base/loading.jl

```diff
@@ -591,9 +591,15 @@ function explicit_manifest_uuid_path(project_file::String, pkg::PkgId)::Union{No
             return path
         end
         hash === nothing && return nothing
+        # Look for vendored packages in `$project/packages`
```

Comment:
```suggestion
        # Look for bundled packages in `$project/packages`
```

Reply:
Is "vendored" the right word here? To me, a vendored dependency is what Pkg does with TOML.

I would say what we are doing here is "bundled" dependencies.

Reply:
Sure, that's a good term for it 👍